assignment/scripts/array-practice.js

```diff
@@ -1,94 +1,106 @@
-console.log('****** Array Practice *******');
+console.log("****** Array Practice *******");
```

Comment:
Generally speaking you want to reign in auto-formatters a bit. They should be configured on a per-project or per-team basis so your code base doesn't end up with a bunch of random changes like this.